_config.yml

```diff
@@ -435,6 +435,9 @@ navigation:
                   - text: Acquisition engagement glossary
                     url: working-on-an-acquisition-engagement/#acquisition-glossary
                     internal: true
+              - text: Microrequests
```

Comment:
Minor, but I'm wondering if we should have something like "MIcrorequests (labs)" while people get used to the admittedly sorta arcane nomenclature. Thoughts?